### ISSUE TITLE
Double Percent sign  in Query

### ISSUE THREAD

**Issue by michail1982** [yajra/laravel-datatables#665]
### Summary of problem or feature request

Percent sign added several times in one instance
example

### Code snippet of problem
```php
//Search [ 'regex' => false, 'value' => 'SomePeople' ]
$builder = Users::query();
return Datatables:of($builder)->make(true);
```
Expect: users where name like  '%SomePeople%'
DEbugger :   users where name like  '%%SomePeople%%'

Explain 
Used Eloquent Engine; 
Process https://github.com/yajra/laravel-datatables/blob/6.0/src/Engines/BaseEngine.php#L192
then on column processing https://github.com/yajra/laravel-datatables/blob/6.0/src/Engines/QueryBuilderEngine.php#L287

suggest change https://github.com/yajra/laravel-datatables/blob/6.0/src/Engines/QueryBuilderEngine.php#L286
```php
if ($this->isSmartSearch()) {
```
to
```php
if (!$this->isSmartSearch()) {
```
- Operating System CentOs/Windows
- PHP Version 5.6.23/5.6.12
- Laravel Version 5.2.41
- Laravel-Datatables Version 6.16.0

**Comment by yajra** [comment]
@michail1982 thanks for the feedback. Can you please submit a PR if you can?

**Comment by michail1982** [comment]
Done

**Comment by yajra** [comment]
@michail1982, I just reviewed your PR and this issue and I found a better way to fix this. Thanks for the help!

**Comment by michail1982** [comment]
tnx, @yajra

**Comment by yajra** [comment]
Welcome, patch was released on v6.16.1
